README.md

```diff
@@ -76,21 +76,21 @@ _choose one_ of the following submodule-cloning techniques:
 
 - Clone the repo and its submodule at the same time
   using the `--recurse-submodules` option:
-  ```bash
+  ```terminal
```

Comment:
The ones in README and other markdown files not in `/src` should likely continue to use `bash`.  I don't believe `terminal` is a tag supported by GitHub's syntax highlighter.

Reply:
It is.

![image](https://github.com/dart-lang/site-www/assets/706219/095a9a18-7ede-4d61-98cb-14ee4d3064aa)

Reply:
I mean it will still render as a code block, but is not syntax highlighted. You can find a community-maintained list of supported languages on GitHub at https://github.com/jincheng9/markdown_supported_languages

Reply:
Well, that's judgment call then. It doesn't need syntax highlighting here, so stet.